### PR TITLE
 In Python, identity is not the same thing as equality

### PR DESCRIPTION
**Pull request by cclauss**
[flake8](http://flake8.pycqa.org) testing of https://github.com/pybee/yorkshire4 on Python 3.7.1

$ __flake8 . --count --select=E9,F63,F72,F82 --show-source --statistics__
```
./docs/books/computer-games/computer-space-games/beat_the_bug_eyes/beat_the_bug_eyes.py:52:24: F821 undefined name 'getch'
        selected = int(getch(timeout=1))
                       ^
./docs/books/adventure-games/adventure-programs/adventure.py:887:100: F632 use ==/!= to compare str, bytes, and int literals
    if is_active(OBJ_BATS) and is_in_room((1, 5), current_location) and not verb == VERB_SPRAY and random.randint(1, 3) is not 3:
                                                                                                   ^
./docs/books/adventure-games/adventure-programs/adventure.py:892:75: F632 use ==/!= to compare str, bytes, and int literals
    elif is_in_room((5, 4), current_location) and not is_active(DOWN) and random.randint(1, 2) is 1:
                                                                          ^
2     F632 use ==/!= to compare str, bytes, and int literals
1     F821 undefined name 'getch'
3
```

### CODE REVIEW

File: docs/books/adventure-games/adventure-programs/adventure.py [hunk 1]
```diff
@@ -884,12 +884,12 @@ def verb_score(verb, current_location, obj=None):
 
     print(message)
 
-    if is_active(OBJ_BATS) and is_in_room((1, 5), current_location) and not verb == VERB_SPRAY and random.randint(1, 3) is not 3:
+    if is_active(OBJ_BATS) and is_in_room((1, 5), current_location) and not verb == VERB_SPRAY and random.randint(1, 3) != 3:
```
Comment (brutusthebee) [inline docs/books/adventure-games/adventure-programs/adventure.py]: At column 80: (E501) line too long (125 > 79 characters)

File: docs/books/adventure-games/adventure-programs/adventure.py [hunk 2]
```diff
@@ -884,12 +884,12 @@ def verb_score(verb, current_location, obj=None):
...
         # bats are active in Room 13 - Rear Turret Room
         activate_flag(OBJ_BATS)
         message = 'BATS ATTACKING!!'
         print(message)
-    elif is_in_room((5, 4), current_location) and not is_active(DOWN) and random.randint(1, 2) is 1:
+    elif is_in_room((5, 4), current_location) and not is_active(DOWN) and random.randint(1, 2) == 1:
```
Comment (brutusthebee) [inline docs/books/adventure-games/adventure-programs/adventure.py]: At column 80: (E501) line too long (100 > 79 characters)

File: docs/books/adventure-games/adventure-programs/adventure.py [hunk 3]
```diff
@@ -884,12 +884,14 @@ def verb_score(verb, current_location, obj=None):
 
     print(message)
 
-    if is_active(OBJ_BATS) and is_in_room((1, 5), current_location) and not verb == VERB_SPRAY and random.randint(1, 3) is not 3:
+    if (is_active(OBJ_BATS) and is_in_room((1, 5), current_location) and
+        verb != VERB_SPRAY and random.randint(1, 3) != 3):
```
Comment (brutusthebee) [inline docs/books/adventure-games/adventure-programs/adventure.py]: At column 9: (E129) visually indented line with same indent as next logical line

**Comment by cclauss** [comment]
$ __python__
```python
>>> raw = 'ra'
>>> raw += 'w'
>>> raw
'raw'
>>> raw == 'raw'
True
>>> raw is 'raw'
False
>>> 0 == 0.0
True
>>> 0 is 0.0
False
```

**Pull request closed without merging**